src/kubeconfig.ts

```diff
@@ -2,12 +2,14 @@
 
 import { shell } from './shell';
 import { fs } from './fs';
+import * as vscode from 'vscode';
 import * as yaml from 'js-yaml';
 
 export function readKubectlConfig(): Promise<KubeConfig> {
     return new Promise((resolve, reject) => {
-        const kubeConfig = shell.combinePath(shell.home(), ".kube/config");
+        const kubeConfig = shell.combinePath(shell.home(), '.kube/config');
         fs.readFile(kubeConfig, 'utf8', (err, data) => {
+            console.log(data);
```

Comment:
I don't think we need to add the `console.log` statement, or is it just being moved from elsewhere?

Reply:
fixed.